features/create_articles.feature

```diff
@@ -11,18 +11,19 @@ Feature: Create articles
     Then I fill in "Title" with "A Whole New World"
     And I fill in "Content" with "A new fantastic point of view"
     And I click "Create Article" button
-    Then I should be redirected to Homepage
+    Then I should be on "A Whole New World" page
+    And I should see "Article was successfully created."
+    And I should see "A Whole New World"
+    And I should see "A new fantastic point of view"
 
   Scenario: Failed to fill in content field
     When I click "New Article" link
     Then I fill in "Title" with "A Whole New World"
     Then I fill in "Content" with " "
     And I click "Create Article" button
-    Then I should stay on new article page
```

Comment:
did you delete this Then line? something needs to happen after you click "create article"